03.asynchronous/callback.js

```diff
@@ -0,0 +1,31 @@
+import sqlite3 from "sqlite3";
+
+const db = new sqlite3.Database(":memory:");
+
+db.run(
+  "CREATE TABLE books (id INTEGER PRIMARY KEY AUTOINCREMENT, title TEXT NOT NULL UNIQUE)",
+  () => {
+    db.run(
+      "INSERT INTO books (title) VALUES (?)",
+      "ゼロからわかるRuby超入門",
+      function () {
+        const lastID = this.lastID;
+        console.log(`ID:${lastID}`);
```

Comment:
見づらいので空白を入れたほうがよさそうです。19 行目についても同様です。

```suggestion
        console.log(`ID: ${lastID}`);
```